src/components/enterprise-user-subsidy/UserSubsidy.jsx

```diff
@@ -44,17 +46,8 @@ const UserSubsidy = ({ children }) => {
       </div>
     );
   }
-
   return (
     <>
-      {/**
-       * SubscriptionSubsidy takes care of redirecting the user to `/${enterpriseConfig.slug}`
-       * if their organization has a subscription plan but they don't have appropriate access
-       * to a license (i.e., status="activated"). it also handles the case where the organization
-       * has an active subscription plan but the current date is not between the plan's start and
-       * expiration dates. The component also handles rendering warning/error status alerts.
-       */}
-      <SubscriptionSubsidy plan={subscriptionPlan} license={subscriptionLicense} />
       {/**
```

Comment:
nit: We can probably remove this comment now (line 51-56) as I'm not sure it's relevant anymore.